packages/core/package.json

```diff
@@ -438,6 +438,7 @@
         "serveVue": "Local server for Vue.js code for development purposes. Provides faster iteration when updating Vue files"
     },
     "devDependencies": {
+        "@aws/language-server-runtimes": "0.2.27",
```

Comment:
Should this live in `packages/amazonq/package.json`  ? I guess not, because toolkit may want to use aws lsp too. Plus it's a "dev" dep so less of a concern.

Reply:
I think we will want this for the notifications language server (I believe that's still happening?)